components/contactPage/contactForm/ContactForm.js

```diff
@@ -0,0 +1,32 @@
+import TextInputField from '../../textInputField/TextInputField';
+import TextInputBox from '../../textInputBox/TextInputBox';
+import SquareButton from '../../squareButton/SquareButton';
+import {
+  ContactFormContainer,
+  InputFieldWrapper,
+  InputBoxWrapper,
+  ButtonWrapper,
+} from './ContactForm.styles';
+export default function ContactForm({ togglePopup }) {
+  return (
+    <>
+      <ContactFormContainer>
+        <InputFieldWrapper>
+          <TextInputField id="1" label="First Name" />
```

Comment:
Could you please just make sure these have more human readable names/ids? The point of them is to let those with screen readers know what they're for. 

https://www.w3.org/WAI/tutorials/forms/